### PR TITLE
reorganized courses grid, added tests

### PR DESCRIPTION
**Pull request by dominikbruechner**
code refactoring + tst

### CODE REVIEW

File: src/components/CoursesGrid.unit.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import { shallowMount } from "@vue/test-utils";
```
Comment (adrianjost) [inline src/components/CoursesGrid.unit.js]: not needed, shallowMount is imported globaly
```suggestion
```

File: src/components/CoursesGrid.vue [hunk 2]
```diff
@@ -0,0 +1,46 @@
+<template>
+	<div class="app">
```
[the remark this answers is not in the document]
Comment (adrianjost) [reply src/components/CoursesGrid.vue]: Everything else looks good.

File: src/components/CoursesGrid.vue [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+			</div>
+		</div>
+	</div>
+</template>
+
+<script>
+import CourseCard from "./CourseCard.vue";
+
+export default {
+	name: "CoursesGrid",
```
Comment (adrianjost) [inline src/components/CoursesGrid.vue]: We don't specify the name anymore. It's autogenerated from the filename and is only needed when defining recursive components.

File: src/components/CoursesGrid.vue (hunk 2 above)
[the remark this answers is not in the document]
Comment (dominikbruechner) [reply src/components/CoursesGrid.vue]: oh I was just copying code, and missed it....

File: src/components/CoursesGrid.vue (hunk 3 above)
Comment (dominikbruechner) [reply src/components/CoursesGrid.vue]: kk

**Pull request merged**